be/src/olap/rowset/segment_v2/bloom_filter_page.cpp

```diff
@@ -0,0 +1,114 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "olap/rowset/segment_v2/bloom_filter_page.h"
+
+#include "util/debug_util.h"
+#include "gen_cpp/segment_v2.pb.h"
+
+namespace doris {
+namespace segment_v2 {
+
+Status BloomFilterPageBuilder::add(const uint8_t* vals, size_t count) {
+    for (int i = 0; i < count; ++i) {
+        if (vals == nullptr) {
+            _bf_builder.add_bytes(nullptr, 1);
+        } else {
+            uint64_t hash = _type_info->hash_code64(vals, DEFAULT_SEED);
+            _bf_builder.add_hash(hash);
+        }
+        
+        if (vals != nullptr) {
+            vals += _type_info->size();
+        }
+        ++_num_inserted;
+        if (_num_inserted >= _block_size) {
+            flush();
+        }
+    }
+    return Status::OK();
+}
+
+Status BloomFilterPageBuilder::flush() {
+    Slice data((char*)_bf_builder.bit_set_data(), _bf_builder.bit_set_data_len() * 8);
+    size_t num = 1;
+    RETURN_IF_ERROR(_page_builder->add((const uint8_t*)&data, &num));
```

Comment:
How about when _page_builder is full?

Reply:
here, I set config's data_page_size to 0 of page_builder, it will no be full.